packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.base.tsx

```diff
@@ -578,23 +571,17 @@ export class CoachmarkBase extends BaseComponent<ICoachmarkProps, ICoachmarkStat
       // Every time the event is triggered we want to
       // setTimeout and then clear any previous instances
       // of setTimeout.
-      this._events.on(
-        window,
-        'resize',
-        (): void => {
-          timeoutIds.forEach(
-            (value: number): void => {
-              clearInterval(value);
-            }
-          );
-
-          timeoutIds.push(
-            this._async.setTimeout((): void => {
-              this._setTargetElementRect();
-            }, 100)
-          );
-        }
-      );
+      this._events.on(window, 'resize', (): void => {
+        timeoutIds.forEach((value: number): void => {
+          clearInterval(value);
+        });
+
+        timeoutIds.push(
+          this._async.setTimeout((): void => {
+            this._setTargetElementRect();
+          }, 100)
+        );
+      });
```

Comment:
All changes in this file are just prettier changes.

Reply:
Ugh, I think this is caused by Prettier 1.18, which was unintentionally introduced as part of the Fabric 7 versioning fixes last week. We can either pin it back to 1.17 or run the script to update everything and make a PR.
https://prettier.io/blog/2019/06/06/1.18.0.html#don-t-breakup-call-expressions-when-the-last-argument-is-an-arrow-function-with-a-simple-return-type-6106-by-brainkim